pkg/sdk/testint/security_integrations_gen_integration_test.go

```diff
@@ -1286,10 +1286,11 @@ func TestInt_SecurityIntegrations(t *testing.T) {
 		require.NoError(t, err)
 
 		// check setting empty comment because of lacking UNSET COMMENT
+		// TODO(SNOW-1461780): change this in UNSET
```

Comment:
nit pick: `change this to UNSET` or `replace this with UNSET`

Reply:
Done.